src/plugins/irc-events/message.js

```diff
@@ -91,6 +91,9 @@ module.exports = function(irc, network) {
 		});
 		chan.pushMessage(client, msg, !self);
 
-		LinkPrefetch(client, chan, msg);
+		// No prefetch URLs unless are simple messages.
+		if (data.type === Msg.Type.MESSAGE) {
```

Comment:
I would add `Msg.Type.ACTION` here. If strong opinion against this, I'll change my mind, but as a user I'd expect that.